System/RTETools.cpp

```diff
@@ -3,6 +3,22 @@
 
 namespace RTE {
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	float GetMPP() { return c_MPP; }
```

Comment:
IIRC because these methods don't belong to a class if you define them in the header you end up with unresolved symbols or some other linker error, that's why all the methods are defined in the cpp here.

Reply:
Ahhh right, makes sense. Cool.